claims.go

```diff
@@ -122,12 +130,12 @@ func validateBoundClaims(logger log.Logger, boundClaimsType string, boundClaims,
 
 		actVals, ok := normalizeList(actValue)
 		if !ok {
-			return fmt.Errorf("received claim is not a string or list: %v", actValue)
+			return fmt.Errorf("received claim is not a string, bool, int or list: %v", actValue)
```

Comment:
in this (and the next) error message I decided to use `int` instead of `json.Number` because this error prints to terminal and I think from user perspective `int` is more clear. Not sure what you think.

Reply:
Makes sense to me. The types being `json.Number` is an internal detail 👍